demo/src/main/java/com/braintreepayments/api/demo/DemoFragment.java

```diff
@@ -79,10 +101,14 @@ private void startBrowserSwitch(@Nullable JSONObject metadata) {
         }
     }
 
-    private Uri buildBrowserSwitchUrl() {
+    private Uri buildBrowserSwitchUrl(boolean isAppLink) {
         String url = "https://braintree.github.io/popup-bridge-example/" +
-                "this_launches_in_popup.html?popupBridgeReturnUrlPrefix=" +
-                getReturnUrlScheme() + "://";
+            "this_launches_in_popup.html?";
```

Comment:
qq: why do we need to concatenate these two strings?

Reply:
I think the single string might have reached the character line limit. I thought it'd be more readable to keep the `"this_launches_in_popup.html?"` portion together.